database/migration/migrateData.ts

```diff
@@ -600,6 +600,59 @@ const migrateData = async () => {
         'RE_ASSIGN'
       );
     `)
+
+    console.log(' - Remove any illegal (i.e. overlapping sections) assignment state')
+    await DB.changeSchema(`
+      UPDATE public.review_assignment
+      SET status='ASSIGNED',
+      assigned_sections = assigned_sections, \
+      trigger = 'DEV_TEST';
+    `)
```

Comment:
Would this be setting one review_assignment assigned per existing application in current review level?
What happens if there is already another review_assignment previously assigned? I imagine that would be persisted. Correct?

Reply:
It just makes sure that there's only one assignment per section per level. Where there are overlaps, it'll just keep the first one it finds (which would presumably be by id).